File: java/052624_B4_17010.java
/*
 * https://www.acmicpc.net/problem/17010
 */
import java.io.*;

class Main {
	public static void main(String[] args) throws IOException {
		try (var br = new BufferedReader(new InputStreamReader(System.in))) {
			int L = Integer.parseInt(br.readLine());
			var sb = new StringBuilder();
			while(L-- > 0)
			{
				String[] Nx = br.readLine().split(" ");
				sb.append(Nx[1].repeat(Integer.parseInt(Nx[0]))).append("\n");
			}
			System.out.printf("%s", sb.toString());
		}
	}
}

File: cshap/052624_B4_17010.cs
/*
 * https://www.acmicpc.net/problem/17010
 */
#nullable disable
using System;
using System.Text;

class B4_17010
{
	//static void Main()
	//{
	//	Console.WriteLine("--- Started ---");
	//	Solution();
	//	Console.WriteLine("--- Finished ---");
	//}
	
	static void Solution()
	{
		var sb = new StringBuilder();
		int L = int.Parse(Console.ReadLine());
		for (int i = 0; i < L; i++)
		{
			string[] Nx = Console.ReadLine().Split();
			int N = int.Parse(Nx[0]);
			for (int j = 0; j < N; j++)
			{
				sb.Append(Nx[1]);
			}
			sb.Append('\n');
		}
		Console.WriteLine(sb.Remove(sb.Length-1, 1));
	}
}
